Pyrado/pyrado/tasks/reward_functions.py

```diff
@@ -246,6 +246,8 @@ def __call__(self, err_s: np.ndarray, err_a: np.ndarray, remaining_steps: int =
 class ScaledExpQuadrErrRewFcn(QuadrErrRewFcn):
     """ Reward function that returns the exp of the scaled weighted sum of squared errors """
 
+    c_max: Optional[float]
```

Comment:
c_max is mast not be None as far as I remember

Reply:
It's set to None in the constructor, that's why I added the Optional there – otherwise None would be disallowed. But I can remove it again, the typing didn't help anyway, I had to ignore stuff down below anyway.

Reply:
yes, please remove it again. it should crash if it is none here